lib/src/warnings.dart

```diff
@@ -499,10 +499,13 @@ class PackageWarningCounter {
 
   @override
   String toString() {
-    var errors = '$errorCount ${errorCount == 1 ? "error" : "errors"}';
-    var warnings =
-        '$warningCount ${warningCount == 1 ? "warning" : "warnings"}';
-    return [errors, warnings].join(', ');
```

Comment:
this seems like it could be replaced with:  `return '$errors, $warnings';`.

Reply:
Sure, this case definitely didn't benefit a lot from the change. Switched to that!